addons/mrp/models/stock_move.py

```diff
@@ -132,7 +132,7 @@ class StockMove(models.Model):
         'Done', compute='_compute_is_done',
         store=True,
         help='Technical Field to order moves')
-    order_finished_lot_ids = fields.Many2many('stock.lot', string="Finished Lot/Serial Number", compute='_compute_order_finished_lot_ids')
+    order_finished_lot_id = fields.Many2one('stock.lot', string="Finished Lot/Serial Number", related="raw_material_production_id.lot_producing_id", store=True)
```

Comment:
@amoyaux does it really needs to be stored?

Reply:
@KangOl Yep it's used by a group by in a view. However the rename_fields seems to do the jobs